apps/content/content/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2020 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>com.adobe.dx</groupId>
+        <artifactId>content.reactor</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>content-content</artifactId>
+    <packaging>content-package</packaging>
+
+    <name>${module.name} - Content Package</name>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
```

Comment:
i would consider this zip package an artifact we want to release & deploy

Reply:
Content without apps usually doesn't make sense. And it is embedded into apps. So in dexter we exposed only apps package which has it's own filters embeds content and bundle. Let me know if you still want this changed.

Reply:
Also, @auniverseaway , I added this content package only as a placeholder so that the integration is easier for you if you needed to add anything outside `/apps` and `libs`. Once you are done with the integration if you don't see yourself adding anything in content, then this package/module can be deleted until required again in the future.

Reply:
well, if at some point you deploy the full app version x.y.z, and there is "just" a small problem with content, with what i propose, you could patch and release a x.y.z+1 content package which is a smaller deal to deploy than the full app

Reply:
Done.